packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -477,6 +478,25 @@ async function handler(
     }
   }
 
+  const availableSlots = await getAvailableSlots({
+    input: {
+      startTime: reqBody.start,
+      endTime: reqBody.end,
+      eventTypeId: eventTypeId,
+      eventTypeSlug: eventTypeSlug,
+      timeZone: reqBody.timeZone,
+      usernameList: dynamicUserList,
```

Comment:
Why did you use dynamicUserList? It only has users which are added as users in the url for dynamic events?